src/modules/iasql_functions/rpcs/iasql_create_review.ts

```diff
@@ -0,0 +1,97 @@
+import { IasqlFunctions } from '..';
+import { RpcInput } from '../..';
+import {
+  Context,
+  PostTransactionCheck,
+  PreTransactionCheck,
+  RpcBase,
+  RpcResponseObject,
+} from '../../interfaces';
+
+/**
+ * Method that generates creates the markdown to be shared or used in any version control system
```

Comment:
Nit: "generates creates" is redundant, should go with one of the two verbs.

Reply:
indeed, copy&paste error